src/helpers.js

```diff
@@ -168,6 +168,22 @@ module.exports = function(Chart) {
 			.slice(0, 1)[0]; // return only the top item
 	}
 
+	var doesSupportPassiveEvent = (function() {
```

Comment:
let's call this `supportsEventListenerOptions` for consistency with the main repo

maybe also add a comment:
```
/**
 * Copied from https://github.com/chartjs/Chart.js/blob/4f722ab619be071e90dd9ea67db27032dc9edd2a/src/platforms/platform.dom.js#L109
 */
```

Reply:
OK, no problem. Just I like to do question variable for boolean value